speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backendUnitTests/mocks/SQLDatabaseStorageMock.h

```diff
@@ -10,36 +10,29 @@
 class SQLDatabaseStorageMock : IDataStorage
 {
 public:
-    SQLDatabaseStorageMock()
+    SQLDatabaseStorageMock() {}
+
+    SQLDatabaseStorageMock(tDataToStore p_dataToStore)
+        : SQLDatabaseStorageMock() {}
+
+    void Run(const tBufferPaths& p_bufferPaths)
     {
-        InputFilePath = "";
+        const auto bufferPaths = new tBufferPaths(p_bufferPaths);
+
+        VariableStore::GetInstance().Variables[0] = static_cast<void*>(bufferPaths);
     }
-    void Run(const filesystem::path& p_inputFilePath)
-    {
-        InputFilePath = p_inputFilePath;
 
-        const auto path = new filesystem::path(p_inputFilePath);
+    // This function definition is needed for building (IDataStorage is abstract), but will not be used and therefore not covered.
+    bool StoreData(const tBufferPaths& p_bufferPaths) override  //@NOCOVERAGE
+    {                                                           //@NOCOVERAGE
+        return true;                                            //@NOCOVERAGE
+    }                                                           //@NOCOVERAGE
```

Comment:
Description should be after @nocoverage, also I think it should be possible to use 1 @nocoverage instead of 4
```suggestion
    bool StoreData(const tBufferPaths& p_bufferPaths) override  //@NOCOVERAGE, This function definition is needed for building (IDataStorage is abstract), but will not be used and therefore not covered.
    {                                                           //@NOCOVERAGE
        return true;                                            //@NOCOVERAGE
    }                                                           //@NOCOVERAGE
```